examples/webgl_loader_imagebitmap.html

```diff
@@ -76,50 +150,30 @@
 
 				group.add( new THREE.GridHelper( 4, 12 ) );
 
-				new THREE.ImageBitmapLoader()
-				.setOptions( { imageOrientation: 'none' } )
-				.load ( 'textures/planets/earth_atmos_2048.jpg', function( imageBitmap ) {
-
-					var tex = new THREE.Texture ( imageBitmap );
-					tex.needsUpdate = true;
+				cubes = new THREE.Group();
+				scene.add( cubes );
 
-					var cube = new THREE.Mesh(
-						new THREE.BoxBufferGeometry( 1,1,1 ),
-						new THREE.MeshBasicMaterial( {
-							map: tex
-						})
-					);
-					cube.position.set( 1, 0.5, 0 );
-					group.add( cube );
+				addImageBitmap();
+				addImage();
 
-				}, function( p ) {
-					console.log( p );
-				}, function( e ) {
-					console.log( e );
+				var ibBtn = document.getElementById( 'add_ib_btn' );
+				ibBtn.addEventListener( 'click', function( e ) {
+					addImageBitmap();
+					e.preventDefault();
 				} );
 
-				new THREE.ImageBitmapLoader()
-				.setOptions( { imageOrientation: 'flipY' } )
-				.load ( 'textures/planets/earth_specular_2048.jpg', function( imageBitmap ) {
-
-					var tex = new THREE.Texture ( imageBitmap );
-					tex.needsUpdate = true;
-
-					var cube = new THREE.Mesh(
-						new THREE.BoxBufferGeometry( 1,1,1 ),
-						new THREE.MeshBasicMaterial( {
-							map: tex
-						})
-					);
-					cube.position.set( -1, 0.5, 0 );
-					group.add( cube );
-
-				}, function( p ) {
-					console.log( p );
-				}, function( e ) {
-					console.log( e );
+				var iBtn = document.getElementById( 'add_i_btn' );
+				iBtn.addEventListener( 'click', function( e ) {
+					addImage();
+					e.preventDefault();
 				} );
 
+				var clearBtn = document.getElementById( 'clear_btn' );
+				clearBtn.addEventListener( 'click', function( e ) {
+					while( cubes.children.length ) {
+						cubes.remove( cubes.children[ 0 ] );
```

Comment:
Nothing in mind, no. I wasn't aware there was a .dispose() method, I'll add that.